sdk/schemaregistry/azure-data-schemaregistry-apacheavro/src/main/java/com/azure/data/schemaregistry/apacheavro/SchemaRegistryApacheAvroSerializer.java

```diff
@@ -140,13 +137,15 @@ public final class SchemaRegistryApacheAvroSerializer {
      * @param avroSerializer Serializer implemented using Apache Avro.
      * @param serializerOptions Options to configure the serializer with.
      */
-    SchemaRegistryApacheAvroSerializer(SchemaRegistryAsyncClient schemaRegistryClient,
-        AvroSerializer avroSerializer, SerializerOptions serializerOptions) {
-        this.schemaRegistryClient = Objects.requireNonNull(schemaRegistryClient,
-            "'schemaRegistryClient' cannot be null.");
-        this.avroSerializer = Objects.requireNonNull(avroSerializer,
-            "'avroSerializer' cannot be null.");
-        this.serializerOptions = Objects.requireNonNull(serializerOptions, "'serializerOptions' cannot be null.");
+    SchemaRegistryApacheAvroSerializer(SchemaRegistryAsyncClient schemaRegistryClient, AvroSerializer avroSerializer,
+        SerializerOptions serializerOptions) {
+
+        Objects.requireNonNull(schemaRegistryClient, "'schemaRegistryClient' cannot be null.");
+        Objects.requireNonNull(serializerOptions, "'serializerOptions' cannot be null.");
+
+        this.avroSerializer = Objects.requireNonNull(avroSerializer, "'avroSerializer' cannot be null.");
+        this.schemaCache = new SchemaRegistrySchemaCache(schemaRegistryClient, serializerOptions.getSchemaGroup(),
```

Comment:
Is there a way for the user to clear the cache?

Reply:
This was a thing that the feature team wanted to get more data on to see if we needed to. So currently, none of the languages have the ability to clear it.

Reply:
Should we at least make the `maxCacheSize` configurable then? We should have some way for the user to control the size of the cache.

Reply:
I created an issue to follow up in a few months: https://github.com/Azure/azure-sdk-for-java/issues/28872